### DISCUSSION TITLE
posts/nju-program-analysis-a6/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#28] [Announcements]
# posts/nju-program-analysis-a6/

南大软分实验笔记｜A6 上下文敏感的指针分析

https://blog.wohin.me/posts/nju-program-analysis-a6/

**Comment by gulugulubing** [comment]
objecSelector和typeSelector，处理静态调用只要用callsite的上下文就行了，老师作业里提示啦。这个作业感觉细节挺多的，我开始也好多概念有点混了。

**Reply by brant-ruan** [reply]
是的，本身Java的细节就很多，要针对每种情况都覆盖到位，还是需要下功夫的。
